routes/postgit.js

```diff
@@ -0,0 +1,15 @@
+//jshint esversion:8
+const router = require("express").Router();
+const verify = require("./verifyToken");
+const Users = require("../model/User");
+
+router.post("/", verify, async (req, res) => {
+  Users.findByIdAndUpdate(req.user,{ $push: { github: req.body.github } } , err => {
+    if (err) {
+      console.error(err);
+    }
+  });
+  res.send("Pushed the URL of your repository");
```

Comment:
This should be inside the `findByIdAndUpdate` function

Reply:
I have resolved it.